src/tests/test_magic.py

```diff
@@ -1664,3 +1665,35 @@ def test_warning_if_variable_defined_but_named_param_is_quoted(
         match=expected_warning,
     ):
         ip.run_cell(cell)
+
+
+def test_run_cte(ip):
```

Comment:
I think we can add a comment here why this test case is important (to handle case when query is itself a cte)

Reply:
added a clearer [test name](https://github.com/ploomber/jupysql/pull/795/commits/7c749e9d9ce8556fce4ef21c98e76c13b4b7e219)